### PR TITLE
update iptables _after_ we've installed everything

### PR DESCRIPTION
**Pull request by BenTheElder**
/hold
I'm actually building this locally to verify 🙄 
xref: #14812 
/cc @cblecker

### CODE REVIEW

**Comment by cblecker** [comment]
/lgtm
Pending local build/verify :)

File: images/krte/Dockerfile [hunk 1]
```diff
@@ -94,7 +89,12 @@ RUN echo "Installing Packages ..." \
     && apt-get update \
     && apt-get install -y --no-install-recommends docker-ce \
     && rm -rf /var/lib/apt/lists/* \
-    && sed -i 's/cgroupfs_mount$/#cgroupfs_mount\n/' /etc/init.d/docker
+    && sed -i 's/cgroupfs_mount$/#cgroupfs_mount\n/' /etc/init.d/docker \
+    && echo "Ensuring Legacy Iptables ..." \
+    && update-alternatives --set iptables /usr/sbin/iptables-legacy \
+    && update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy \
+    && update-alternatives --set arptables /usr/sbin/arptables-legacy \
```
Comment (BenTheElder) [inline images/krte/Dockerfile]: arptables is not installed, nor is ebtables.

**Comment by BenTheElder** [comment]
succesfully built krte, building bootstrap ...

**Comment by BenTheElder** [comment]
builds all good

**Pull request merged**